pkg/render/istio/resources.go

```diff
@@ -0,0 +1,259 @@
+// Copyright (c) 2025 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package istio
+
+import (
+	"bytes"
+	_ "embed"
+	"fmt"
+	"strings"
+
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	admregv1 "k8s.io/api/admissionregistration/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	autoscalingv2 "k8s.io/api/autoscaling/v2"
+	corev1 "k8s.io/api/core/v1"
+	policyv1 "k8s.io/api/policy/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	IstiodDeploymentName      = "istiod"
+	IstioCNIDaemonSetName     = "istio-cni-node"
+	IstioZTunnelDaemonSetName = "ztunnel"
+)
+
+var (
+	//go:embed base.tgz
+	baseTgz []byte
+	//go:embed istiod.tgz
+	istiodTgz []byte
+	//go:embed cni.tgz
+	cniTgz []byte
+	//go:embed ztunnel.tgz
+	ztunnelTgz []byte
+)
+
+type IstioResources struct {
+	CRDs    []client.Object
+	Base    []client.Object
+	Istiod  []client.Object
+	CNI     []client.Object
+	ZTunnel []client.Object
+
+	IstiodDeployment *appsv1.Deployment
+	CNIDaemonSet     *appsv1.DaemonSet
+	ZTunnelDaemonSet *appsv1.DaemonSet
+}
+
+// GetResources returns istio-chart generated templates
+func GetResources(namespace, releaseName string, baseOptions, istiodOptions, cniOptions,
+	ztunnelOptions map[string]interface{}) (*IstioResources, error) {
```

Comment:
```suggestion
func GetResources(namespace, releaseName string, baseOptions, istiodOptions, cniOptions, ztunnelOptions map[string]interface{}) (*IstioResources, error) {
```

Either all on one line, or each argument should get its own line. Don't think this one is long enough to warrant splitting.